.github/workflows/deployTest.yml

```diff
@@ -36,6 +36,10 @@ jobs:
       - uses: azure/login@v1
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
+      - name: Install postgres so that terraform can execute psql
```

Comment:
We don't actually execute psql in this job, though, do we? All of this is currently manual-only.

Reply:
Terraform does, to create the actual user